### DISCUSSION TITLE
is there any way to set channel status???

### DISCUSSION THREAD

**Discussion by Halfsell** [discordjs/discord.js#10083] [Q&A]
first time using discord.js and could not find a way to set/change channel status
![이미지_2024-01-13_044247941](https://github.com/discordjs/discord.js/assets/136581335/2bcdda93-6a69-49ea-90c9-eeee0c973e9d)

**Comment by sdanialraza** [comment]
It hasn't been documented by Discord yet: https://github.com/discordjs/discord.js/issues/10079
